pkg/pkgmgr/rpm.go

```diff
@@ -113,15 +113,32 @@ func isLessThanRPMVersion(v1, v2 string) bool {
 // Map the target image OSType & OSVersion to an appropriate tooling image.
 func getRPMImageName(manifest *unversioned.UpdateManifest, osType string, osVersion string, useCachePrefix bool) string {
 	var image, version string
+	imagePrefix := "mcr.microsoft.com"
+	useCache := useCachePrefix
 
-	if osType == utils.OSTypeAzureLinux {
+	switch {
+	case osType == utils.OSTypeAzureLinux:
 		image = "azurelinux/base/core"
 		if strings.Contains(osVersion, "3.0") {
 			version = "3.0"
 		} else {
 			version = osVersion
 		}
-	} else {
+	case utils.IsSUSEImage(osType):
+		imagePrefix = "registry.opensuse.org"
+		version = osVersion
+		useCache = false
+
+		switch osType {
+		case "sles":
+			imagePrefix = "registry.suse.com"
+			image = "bci/bci-base"
+		case "opensuse-leap":
+			image = "opensuse/leap"
+		case "opensuse-tumbleweed":
```

Comment:
should we use the string constants defined in `pkg/utils/os.go` here?

Reply:
I added the images to the mirror workflow. PTAL.